web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -136,6 +165,11 @@ public function getDefaultParamValue($type, $params) {
         $defaultParam = $paramsDecoded['filters']['types'][0];
         break;
 
+      case 'tags':
+        $tid = (int) $paramsDecoded['filters']['tags'][0];
```

Comment:
I have a feeling that this method will change a lot in the near future as we make the data more complex. So I don't want to overthinking it. 

The only think we may want to change right now is avoiding warnings around `$paramsDecoded['filters']['types'][0]`. It is possible that these keys will not exist. (Okay it is not really possible since this tool is not live, but I'm approaching this from a future-proof point of view.) So we may want to check if !empty($paramsDecoded['filters']['types'][0]) to avoid "illegal offset" and "Undefined array key" warnings.

Reply:
Good call. Fixed.